platform/jobs/edxPlatformQualityDiff.groovy

```diff
@@ -113,6 +116,10 @@ jobConfigs.each { jobConfig ->
             }
             timestamps()
             colorizeOutput('gnome-terminal')
+            preBuildCleanup {
+                includePattern(deleteReports)
```

Comment:
Am I reading this right? We 'include' the following paths in the 'delete'?

Reply:
Yes, counter-intuitively enough: https://jenkinsci.github.io/job-dsl-plugin/#method/javaposse.jobdsl.dsl.helpers.wrapper.WrapperContext.preBuildCleanup

> Makes cleanup more selective by specifying file to be deleted using Ant syntax.